core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -176,6 +176,7 @@ public void testRpmContents()
             assertThatPaths(files)
```

Comment:
Can we have similar test for `DockerIT`?

Reply:
We could have something similar to it

Reply:
`DockerIT` test is not available but we could work on something similar.